hod/local.py

```diff
@@ -76,7 +79,7 @@ def known_cluster_labels():
     """
     Return list of known cluster labels.
     """
-    path = cluster_config_dir()
+    path = cluster_info_dir()
     if os.path.exists(path):
```

Comment:
Do we need to hit the beleaguered FS twice? Can we call `listdir` and catch OSError or whatever falls out? It's only 1 drop out of the avalanche of filesystem harassment that is Python so I don't think it matters. But it probably doesn't matter. :disappointed: